client/src/lib/stores/layout/title.ts

```diff
@@ -0,0 +1,13 @@
+import { writable, derived } from "svelte/store";
+import { Maybe } from "$lib/util/maybe";
+import { siteTitle } from "$lib/site";
+
+export const siteSection = writable<Maybe<string>>(null);
+
+export const pageTitle = derived(siteSection, (siteSectionValue): string => {
```

Comment:
je me questionne sur l'usage d'un store ici. Pourquoi ne pas mettre en dur directement le titre dans le code ?

Reply:
Tu veux parler d'écrire un `<svelte:head><title>...</title></svelte:head>` sur toutes les pages ?

J'ai ajouté quelques mots de motivation générale à la PR

Reply:
Il y a deux choses dans ta réponse :sweat_smile: 

1. L'usage du store

Je remarque que tu suis le pattern suivant :

1. Définir une variable que tu injecte dans un store dans le ficher : client/src/routes/(app)/contribuer/+page.ts
2. Récupérer cette variable dans client/src/routes/(app)/contribuer/+page.svelte

Ma question est: pourquoi utiliser un store dans ce cas ? Pourquoi ne pas directement écrire le titre en dur ?

2. Par rapport au besoin :

Il fait carrément sens pour moi. J'irais même plus loin il est nécessaire au regard du RGAA.

Avoir un tirre par page (au sens `<title> </title>`) correspond aux test suivants du RGAA : 

- **Chaque page web a-t-elle un titre de page ?**
- **Pour chaque page web ayant un titre de page, ce titre est-il pertinent ?**

Durant la formation avec access 42, j'ai noté :

Le `<title>` reprend souvent le titre principal du contenu et le nom du site. De plus, 
- il faut le rendre explicite lors de la mise à jour du contenu de manière dynamique
- en cas de recherche, il faut indiquer la pagination et si possible le terme rechercher
- en cas de soummision de formulaire, il faut indiquer si il y a des erreurs

Bref, ici je comprends bien le besoin mais un peu moins le choix d'implem

Reply:
> Pourquoi ne pas directement écrire le titre en dur ?

Donc, tu confirmes, tu suggèrerais de mettre un `<svelte:head>` directement dans chaque `+page.svelte` ? Je ne suis peut-être pas clair sur l'envie de ne donner aux pages que le contrôle sur le `<Titre de section>`, en ayant un format de titre défini au niveau du layout racine.

Reply:
> Donc, tu confirmes, tu suggèrerais de mettre un <svelte:head> directement dans chaque +page.svelte ?

Oui :-)

Reply:
Tu as approuvé du coup je ne comprends pas trop. C'est OK comme ça ? Mon raisonnement t'a convaincu ?